packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -136,7 +137,9 @@ class AnalysisServer {
   void _handleAnalysisIssues(Map<String, dynamic> issueInfo) {
     // {"event":"analysis.errors","params":{"file":"/Users/.../lib/main.dart","errors":[]}}
     final String file = issueInfo['file'];
-    final List<AnalysisError> errors = issueInfo['errors']
+    final List<dynamic> errorsList = issueInfo['errors'];
+    final List<AnalysisError> errors = errorsList
+        .map(castStringKeyedMap)
```

Comment:
The two `map` calls should have type parameters `Map<String, dynamic>` and `AnalysisError`, respectively.

Reply:
added